src/Annotations/AbstractAnnotation.php

```diff
@@ -137,7 +137,6 @@ public function __set($property, $value)
             unset($fields[$_property]);
         }
         Logger::notice('Unexpected field "' . $property . '" for ' . $this->identity() . ', expecting "' . implode('", "', array_keys($fields)) . '" in ' . $this->_context);
```

Comment:
I think we'll try this and see if anyone complains.
Could we just slightly change the log message to make it clear what is happening?
```suggestion
        Logger::notice('Ignoring unexpected field "' . $property . '" for ' . $this->identity() . ', expecting "' . implode('", "', array_keys($fields)) . '" in ' . $this->_context);
```

Reply:
I might actually pull this change into `master` too...

Reply:
https://github.com/zircote/swagger-php/pull/1476